services/grafana-logging/6.60.2/defaults/cm.yaml

```diff
@@ -7,9 +7,8 @@ data:
   values.yaml: |
     ---
     priorityClassName: "dkp-critical-priority"
-    # pinning grafana to 8.5 since it appears that v9+ is causing the kubernetes audit dashboard to crash
     image:
-      tag: 8.5.26
+      tag: 10.4.2
```

Comment:
what is the default version of grafana shipped with this helm chart? can we just use the default instead of overriding?

You'll have to make this same change to `project-grafana-logging` default values as well

Reply:
yes. We need to make the same change to project-grafana-logging.